docs/src/content/ignition/docs/guides/upgradeable-proxies.md

```diff
@@ -0,0 +1,397 @@
+# Upgradeable proxies
```

Comment:
```suggestion
# Upgradeable Contracts
```
What do we think? I know I proposed "Upgradeable Proxies", but I think contracts is more comprehensible than proxies.